snuba/api.py

```diff
@@ -222,22 +272,14 @@ def parse_and_run_query(dataset, validated_body, timer):
             if len(exclude_group_ids) > max_group_ids_exclude:
                 final = True
             else:
-                where_conditions.append(('group_id', 'NOT IN', exclude_group_ids))
+                body['conditions'].append(('group_id', 'NOT IN', exclude_group_ids))
     else:
         final = False
+        if 'sample' not in body:
+            body['sample'] = settings.TURBO_SAMPLE_RATE
 
-    if final:
-        from_clause = u'{} FINAL'.format(from_clause)
-
-    sample = body.get('sample', settings.TURBO_SAMPLE_RATE if turbo else config_sample)
-    if sample != 1:
-        from_clause = u'{} SAMPLE {}'.format(from_clause, sample)
-
-    joins = []
-
-    if 'arrayjoin' in body:
-        joins.append(u'ARRAY JOIN {}'.format(body['arrayjoin']))
-    join_clause = ' '.join(joins)
+    if 'sample' not in body and config_sample is not None:
+        body['sample'] = config_sample
```

Comment:
I'm not sure how helpful this configuration parameter is, since it changes the response so drastically. I'd like to remove it but this preserves the existing behavior for now.